### ISSUE TITLE
Very poor streaming for openvidu-mvc-java tutorial on aws.

### ISSUE THREAD

**Issue by philajay** [OpenVidu/openvidu#136]
I installed opnvidu using cloudformation on **t2.large ec2 instance**. On another **t2.medium** ec2 instance I installed openvidu tutorials. Then I run the openvidu-mvc-java instance after updating the openViduURL. I joined as publisher on one chrome tab and as a subscriber on another incognito tab.  On the subscriber video streaming is very poor. It seems like it streams for a couple of seconds and then frame freezes.  After some time it will again stream for couple of seconds and again it freezes.

**Comment by micaelgallego** [comment]
This behaviour is very extrange. We tested OpenVidu Cloud Formation and work as expected.

Can you provide us more information? Have you tested with other videoconf apps to see if you have a network problem?

**Comment by philajay** [comment]
So while I was doing the above test with openvidu I also installed kurento server at second machine and used that to stream the video. The idea was to rule out the network problem. The application with the kurento server streamed it perfectly. 

Unfortuanately I deleted the ec2 instance which I used for testing so I do not have the logs. As I dont have any logs and it seems to be working fine at your end I'm closing this issue.
